### PR TITLE
Mqtt service added.

### PR DESCRIPTION
**Pull request by ababsmuc**

### CODE REVIEW

File: LorafyAPI/LorafyAPI.csproj [hunk 1]
```diff
@@ -16,6 +17,7 @@
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
     <PackageReference Include="Microsoft.EntityFrameworkCore.Relational" Version="7.0.0" />
+    <PackageReference Include="MQTTnet" Version="4.1.4.563" />
```
Comment (stijnstroeve) [inline LorafyAPI/LorafyAPI.csproj]: Why have you added both MQTTnet and M2Mqtt?

File: LorafyAPI/appsettings.Development.json [hunk 2]
```diff
@@ -5,6 +5,12 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
+  "MQTT_variables": {
```
Comment (stijnstroeve) [inline LorafyAPI/appsettings.Development.json]: Please add to git ignore :)

File: LorafyAPI/MQTT_BackgroundService.cs [hunk 3]
```diff
@@ -0,0 +1,78 @@
+﻿using Microsoft.Extensions.Logging;
+using System.Diagnostics;
+using System.Threading;
+using uPLibrary.Networking.M2Mqtt;
+using uPLibrary.Networking.M2Mqtt.Messages;
+public class MQTT_BackgroundService : BackgroundService
```
Comment (stijnstroeve) [inline LorafyAPI/MQTT_BackgroundService.cs]: Can you remove the `_` from the class name? Its a convention to use 1 casing at once.

File: LorafyAPI/MQTT_BackgroundService.cs [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+    }
+
+    public MQTT_BackgroundService(ILogger<MQTT_BackgroundService> logger, IConfiguration configuration)
+    {
+        _logger = logger;
+        Configuration = configuration;
+    }
+
+    public override Task ExecuteTask => base.ExecuteTask;
+
```
Comment (stijnstroeve) [inline LorafyAPI/MQTT_BackgroundService.cs:37]: Can you format the document so there is only 1 whitespace in between?

File: LorafyAPI/MQTT_BackgroundService.cs [hunk 5]
```diff
@@ -0,0 +1,78 @@
+﻿using Microsoft.Extensions.Logging;
+using System.Diagnostics;
+using System.Threading;
+using uPLibrary.Networking.M2Mqtt;
+using uPLibrary.Networking.M2Mqtt.Messages;
```
Comment (stijnstroeve) [inline LorafyAPI/MQTT_BackgroundService.cs:5]: Please a newline after the imports

**Review by stefank02 — APPROVED** [review]
LGTM

**Review by stijnstroeve — CHANGES_REQUESTED** [review]

**Review by stefank02 — APPROVED** [review]
LGTM

**Review by stijnstroeve — APPROVED** [review]
LGTM :D

**Pull request merged**